com.chartboost.mediation/Runtime/AdFormats/Banner/Unity/ChartboostMediationUnityBannerAd.cs

```diff
@@ -0,0 +1,227 @@
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Chartboost.Banner;
+using Chartboost.Requests;
+using Chartboost.Results;
+using Chartboost.Utilities;
+using Newtonsoft.Json;
+using UnityEngine;
+using UnityEngine.UI;
+using Logger = Chartboost.Utilities.Logger;
+
+namespace Chartboost.AdFormats.Banner.Unity
+{
+    public enum UnityBannerAdSize
+    {
+        Adaptive,
+        Standard,
+        Medium,
+        Leaderboard
+    }
+    
+    public delegate void ChartboostMediationUnityBannerAdEvent();
+    public delegate void ChartboostMediationUnityBannerAdDragEvent(float x, float y);
+    
+    public partial class ChartboostMediationUnityBannerAd : MonoBehaviour
+    {
+        public ChartboostMediationUnityBannerAdEvent WillAppear;
+        public ChartboostMediationUnityBannerAdEvent DidClick;
+        public ChartboostMediationUnityBannerAdEvent DidRecordImpression;
+        public ChartboostMediationUnityBannerAdDragEvent DidDrag;
+        
+        [SerializeField] 
+        private bool autoLoadOnInit = true;
+        [SerializeField] 
+        private string placementName;
+        [SerializeField] 
+        private bool draggable = true;
+        
+        [SerializeField][HideInInspector] 
+        private UnityBannerAdSize size;
+        [SerializeField][HideInInspector] 
+        private bool resizeToFit;
+        [SerializeField][HideInInspector] 
+        private ChartboostMediationBannerHorizontalAlignment horizontalAlignment = ChartboostMediationBannerHorizontalAlignment.Center;
+        [SerializeField][HideInInspector] 
+        private ChartboostMediationBannerVerticalAlignment verticalAlignment = ChartboostMediationBannerVerticalAlignment.Center;
+        
+        private IChartboostMediationBannerView _bannerView;
+        
+        public override string ToString()
+        {
+            base.ToString();
+            return JsonConvert.SerializeObject(BannerView);
+        }
+
+        # region Unity Lifecycle
+        
+        private void Start() => ChartboostMediation.DidStart += ChartboostMediationOnDidStart;
+
+        private void OnEnable() => BannerView?.SetVisibility(true);
+
+        private void OnDisable() => BannerView?.SetVisibility(false);
+
+        public void OnDestroy() => BannerView?.Destroy();
+        
+        #endregion
+        
+        public string PlacementName
+        {
+            get => placementName;
+            internal set => placementName = value;
+        }
+        public bool Draggable
```

Comment:
```suggestion
        public bool Draggable
        
```

Reply:
I think this was meant to have a blank line _before_ `public bool Draggable` ? 
The blank line is added now